src/styles/components/card.css

```diff
@@ -0,0 +1,105 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  max-width: 200px;
+  padding: 16px;
+  background-color: var(--card-bg);
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 16px auto 0;
+}
+
+.card__title {
+  margin-top: 40px;
+
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--primary);
+}
+
+.card__product-code {
+  margin-top: 4px;
+
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__rating {
+  margin-top: 16px;
+}
+
+.rating {
```

Comment:
that's why you don't need this new block
[checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md) 11th item